fastlane/Fastfile

```diff
@@ -209,29 +372,33 @@ platform :android do
       lint_task: 'lintRelease'
     )
 
+    version = current_release_version
```

Comment:
`version` is already defined on line 365. Do we need this again?

Reply:
Fixed in https://github.com/Automattic/pocket-casts-android/pull/1425/commits/eb2eb1e6659e1c3f930794454b8a626fd7ed0fa6